python/bindings/openravepy_global.cpp

```diff
@@ -1331,6 +1331,36 @@ object TransformLookat(object olookat, object ocamerapos, object ocameraup)
     return toPyArray(transformLookat(ExtractVector3(olookat),ExtractVector3(ocamerapos),ExtractVector3(ocameraup)));
 }
 
+object OrientedBoxFromAABB(object oab, object otransform)
+{
+    return toPyOrientedBox(OrientedBoxFromAABB(ExtractAABB(oab), ExtractTransform(otransform)));
+}
+
+object AABBFromOrientedBox(object oobb)
+{
+    return toPyAABB(AABBFromOrientedBox(ExtractOrientedBox(oobb)));
+}
+
+bool AABBCollision(object oab1, object oab2)
```

Comment:
@eisoku9618 let's also change name of this function. and also do the same to c++. and mark the existing one obsolete

Reply:
@yoshikikanemoto sure, I added `CheckAABBCollision()` and appended deprecated attribute to existing `AABBCollision()`.

Reply:
can we remove `AABBCollision` from python bindings since it's not there before this PR?

Reply:
Yes, thank you!